.dotnet/src/Custom/Common/ServiceLatencyTierOutcome.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace OpenAI;
+
+[CodeGenModel("ChatCompletionServiceTier")]
+public readonly partial struct ServiceLatencyTierOutcome
+{
+    [CodeGenMember("Scale")]
+    public static ServiceLatencyTierOutcome ScaleCreditsApplied { get; } = new(ScaleCreditsAppliedValue);
```

Comment:
If you agree with keeping "Auto", the same logic for keeping "Scale" works here.